src/SprintRun.ts

```diff
@@ -104,9 +127,12 @@ export default class SprintRun {
 	startSprint(previousWordCount : number, update : (status : string, statusChanged : boolean) => void, endOfSprintCallback : (sprintRunStat : SprintRunStat) => void): number {
 		this.endOfSprintCallback = endOfSprintCallback
 		this.previousWordCount = previousWordCount
+		this.wordsLastCount = previousWordCount
 
 		const now = Date.now()
 		this.created = moment.utc().valueOf()
+		this.timeStart = now
+		this.timeEnd = now + this.sprintLengthInMS
```

Comment:
I'd rather we just did
`this.millisecondsLeft = (now + this.sprintLengthInMS) - currentNow`
rather than create a class variable we never use again

Reply:
I just restored the original line of code ```this.millisecondsLeft = this.sprintLengthInMS - this.elapsedMilliseconds```
There was a reason I had made this tweak, but I don't appear to have followed through with whatever that reason one, so the reason is lost to the void. 😆